app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -114,8 +115,74 @@ def assigned
       render json: { topics: serialize_data(topics, AssignedTopicSerializer) }
     end
 
+    def assignments_count
```

Comment:
Let's try to avoid another AJAX request to load the assignments count. We should be able to include it in the group serializer

Something like `add_to_serializer(:group, :assignments_count)` in plugin.rb